gwsurrogate/new/nodeFunction.py

```diff
@@ -128,6 +128,36 @@ def __call__(self, x):
 
             return super(NRHybSur3dq8Fit, self).__call__(mapped_x)
 
+class NRHybSur2dq15Fit(pySurrogateFit):
+    """
+    Evaluates fits for the NRHybSur3dq8 surrogate model.
```

Comment:
Update docs, NRHybSur3dq8 -> NRHybSur2dq15Fit

Reply:
Thanks for catching that.